packages/telegram/src/setTelegramBotWebhook.ts

```diff
@@ -0,0 +1,20 @@
+import { POST } from '@packages/utils/http';
+
+import { getTelegramBaseUrl } from './auth';
+import type { BotToken } from './auth';
+
+/**
+ * Set the webhook for a telegram bot
+ */
+export async function setTelegramBotWebhook({ token, endpoint }: BotToken & { endpoint: string }) {
+  const url = `${getTelegramBaseUrl({ token })}/setWebhook?url=${encodeURIComponent(endpoint)}`;
+
+  await POST(url, undefined, {
+    query: {
```

Comment:
you're putting the query in `url` but also pasing it in here, i'm assuming that we end up either adding two query strings or the POST is just de-duping it for you